docs/content/posts/2025-03-01-release-2.0.0.md

```diff
@@ -0,0 +1,1639 @@
+
+Configuration APIs that takes java objects as arguments are removed from `StreamExecutionEnvironment` and `ExecutionConfig`. They should now be set via `Configuration` and `ConfigOption`. See also the list of [breaking programming APIs](#breaking_programming_apis).
+
+To avoid exposing internal interfaces, User-Defined Functions no longer have full access to `ExecutionConfig`. Instead, necessary functions such as `createSerializer()`, `getGlobalJobParameters()` and `isObjectReuseEnabled()` can now be accessed from `RuntimeContext` directly.
+
+## Misc
+- State Compatibility is not guaranteed between 1.x and 2.x.
+- Java 8 is no longer supported. The minimum Java version supported by Flink now is Java 11.
+- The Per-job deployment mode is no longer supported. Please use the Application mode instead.
+- Legacy Mode of Hybrid Shuffle is removed.
+
+# Release notes
+
+For a comprehensive list of features, improvements, bug-fixes, as well as adjustments to make and issues to check during the upgrading process, please refer to the [release notes](https://nightlies.apache.org/flink/flink-docs-master/release-notes/flink-2.0/).
+
+# List of Contributors
+
+The Apache Flink community would like to thank each one of the contributors that have made this release possible:
+
+Alan, Aleksandr, Alex, AlexYinHan, Alexander, Ammu, Andrei, Andrey, Arkadiusz, Arvid, BoShuai, Brisk, Cancai, Chesnay, Chester, Chris, CuiYanxiang, Danny, David, Dawid, Dian, Dmitriy, Eaugene, Fabian, Feng, Ferenc, Francesco, Gabor, Gantigmaa, Grace, Grzegorz, Gustavo, Hanyu, Hao, Hong, Hyungstler, Jacky, James, Jeyassri, Jiangjie, Jim, Jingsong, Joern, Joery, JunRuiLee, Junrui, Kaitian, Kartikey, Kunni, Kurt, Lajith, Lei, Lorenzo, Luke, Marc, Martijn, Márton, Mate, Matt, Matthias, Mina, Myracle, Paul, Peng, Peter, Piotr, Qingsheng, Ran, Robin, Roc, Roman, Ron, Rui, Ryan, Sam, Samrat, Sergei, Sergey, Sergio, Shengkai, Shuyi, Stefan, Sud0x67, Tamas, Thomas, Timo, Vincent-Woo, Wang, WangQian, Weijie, Wenchao, Wenjun, Xia, Xiangyu, Xintong, Xu, XuHao41, XuShuai, Xuannan, Xuyang, Yanfei, Yi, Yiyu, Yu, Yubin, Yuxin, Zakelly, Zdenek, Zhanghao, Zhen, anupamaggarwal, argoyal2212, auroflow, candaccc, clarax, codenohup, drymatini, dylanhz, fengli, fredia, gongzhongqiang, haishui, huyuanfeng, jectpro7, lexluo09, lincoln, liuyongvs, lvyanquan, lz, mayuehappy, mehdid93, morazow, naferx, nateab, noorall, r-sidd, shalini, simplejason, slankka, sullis, sunxia, sxnan, tison, wangfeifan, xaniasd, xiarui, xincheng.ljr, xuyang, xuzifu666, yinhan.yh, yunfengzhou-hub, zbz, zhangmang, zhaorongsheng, zhengchenyu, zhuanshenbsj1, 余良, 皆非, 林尚泉
```

Comment:
Out of curiosity: is it intended to use only first names here?
Asking since usually there was used either full name or GH username.

Reply:
Nice catch! It's a mistake in my script for generating this list from the log history that only printed the first token of each author name. Fixed it.